pom.xml

```diff
@@ -636,6 +636,63 @@ under the License.
 
 	<profiles>
 
+		<profile>
+			<id>fast</id>
+			<activation>
+				<property>
+					<name>fast</name>
+				</property>
+			</activation>
+			<build>
+				<pluginManagement>
+					<plugins>
+						<plugin>
+							<groupId>org.apache.rat</groupId>
+							<artifactId>apache-rat-plugin</artifactId>
+							<configuration>
+								<skip>true</skip>
+							</configuration>
+						</plugin>
+						<plugin>
+							<groupId>org.apache.maven.plugins</groupId>
+							<artifactId>maven-checkstyle-plugin</artifactId>
+							<configuration>
+								<skip>true</skip>
+							</configuration>
```

Comment:
If we skip this plugin, Will it affect the format or style of the code of people's PR will merge to flink?

Reply:
+1 not skip plugins on tarvis. LGTM make it fast local building.

Reply:
we still verify everything at least once on travis.

Reply:
That's good. I think this works because of ```.travis.yml``` enable it at least. Correct ?

Reply:
got nothing to do `.travis.yml`. The `travis_mvn_watchdog.sh` script uses differet settings for each build, one of which builds all of Flink with all plugins enabled, (so called "misc.")

Reply:
That sounds nice.